test/dotnet.Tests/CommandTests/Pack/PackTests.cs

```diff
@@ -289,5 +290,176 @@ public void DotnetPackAcceptsRuntimeOption()
             result.Should().Fail()
                 .And.HaveStdOutContaining("NETSDK1083");
         }
+        [Fact]
+        public void DotnetPack_AcceptsPropertiesOption()
+        {
+            var testInstance = _testAssetsManager.CopyTestAsset("TestNuspecProject")
+                .WithSource();
+            string nuspecPath = Path.Combine(testInstance.Path, "PackNoCsproj.nuspec");
+            var result = new DotnetPackCommand(Log)
+                .WithWorkingDirectory(testInstance.Path)
+                .Execute(nuspecPath, "--property", "id=CustomID");
+
+            result.Should().Pass();
+
+            var outputDir = new DirectoryInfo(testInstance.Path);
+            outputDir.Should().Exist()
+                .And.HaveFile("CustomId.1.0.0.nupkg");
+
+            var nupkgPath = Path.Combine(testInstance.Path, "CustomId.1.0.0.nupkg");
+            File.Exists(nupkgPath).Should().BeTrue("The package should be created with the custom id");
+
+            using var zip = ZipFile.OpenRead(nupkgPath);
+            var nuspecEntry = zip.Entries.FirstOrDefault(e => e.FullName.EndsWith(".nuspec"));
+            nuspecEntry.Should().NotBeNull("The .nuspec file should exist in the package.");
+
+            using var nuspecStream = nuspecEntry.Open();
+            using var reader = new StreamReader(nuspecStream);
+            var nuspecContent = reader.ReadToEnd();
+
+            nuspecContent.Should().Contain("<id>CustomID</id>");
+        }
+
+        [Fact]
+        public void DotnetPack_AcceptsVersionOption()
+        {
+            var testInstance = _testAssetsManager.CopyTestAsset("TestNuspecProject")
+                .WithSource();
+            string nuspecPath = Path.Combine(testInstance.Path, "PackNoCsproj.nuspec");
+            var result = new DotnetPackCommand(Log)
+                .WithWorkingDirectory(testInstance.Path)
+                .Execute(nuspecPath, "--version", "1.2.3");
+
+            result.Should().Pass();
+
+            var outputDir = new DirectoryInfo(testInstance.Path);
+            outputDir.Should().Exist()
+                .And.HaveFile("PackNoCsproj.1.2.3.nupkg");
+
+            var nupkgPath = Path.Combine(testInstance.Path, "PackNoCsproj.1.2.3.nupkg");
+            File.Exists(nupkgPath).Should().BeTrue("The package should be created with the specified version.");
+
+            using var zip = ZipFile.OpenRead(nupkgPath);
+            var nuspecEntry = zip.Entries.FirstOrDefault(e => e.FullName.EndsWith(".nuspec"));
+            nuspecEntry.Should().NotBeNull("The .nuspec file should exist in the package.");
+
+            using var nuspecStream = nuspecEntry.Open();
+            using var reader = new StreamReader(nuspecStream);
+            var nuspecContent = reader.ReadToEnd();
+
+            nuspecContent.Should().Contain("<version>1.2.3</version>");
```

Comment:
Similar feedback as the previous test. 

https://github.com/NuGet/NuGet.Client/blob/4ce65d4a1c482eda1c8656fc032d1f5cf247763a/test/NuGet.Core.FuncTests/Dotnet.Integration.Test/PackCommandTests.cs#L531-L533.

You can use the NuGet APIs